### PR TITLE
fix(expression-values): allow null values

### PR DESCRIPTION
**Pull request by flipactual**

### CODE REVIEW

File: __tests__/Exciter.js [hunk 1]
```diff
@@ -133,10 +133,10 @@ test('deleteFailReject', (done) => {
   sinon.stub(exciterAllowReject.dynamo, 'delete').callsFake(awsPromiseReject(new Error('Bad thing happened')));
   return exciterAllowReject.delete(pk, 'fake')
     .then(() => {
-      expect('Promise should reject if rejectOnFail is true').toEqual(false);
+      expect('Promise should reject if rejectOnFail is true').toBe(false);
```
Comment (flipactual) [inline __tests__/Exciter.js:136]: There were a lot of unnecessary deep equivalency checks on booleans – hope you don't mind

File: __tests__/Exciter.js [hunk 2]
```diff
@@ -751,7 +751,7 @@ test('normalizeDataValues', (done) => {
       userId: '345',
       nestedMap: { someOther: 'thing' },
     },
-    skipMe: null,
+    skipMe: undefined,
```
Comment (flipactual) [inline __tests__/Exciter.js:754]: I swapped `undefined` in for `null` in several tests to ensure the functionality still works

File: __tests__/Exciter.js [hunk 3]
```diff
@@ -964,15 +964,15 @@ test('valueIsEmpty', (done) => {
...
   // Things that aren't empty.
-  expect(Exciter.valueIsEmpty(1)).toEqual(false);
-  expect(Exciter.valueIsEmpty('not empty')).toEqual(false);
-  expect(Exciter.valueIsEmpty([1, 2])).toEqual(false);
-  expect(Exciter.valueIsEmpty({ not: 'empty' })).toEqual(false);
+  expect(Exciter.valueIsEmpty(1)).toBe(false);
+  expect(Exciter.valueIsEmpty('not empty')).toBe(false);
+  expect(Exciter.valueIsEmpty([1, 2])).toBe(false);
+  expect(Exciter.valueIsEmpty({ not: 'empty' })).toBe(false);
+  expect(Exciter.valueIsEmpty(null)).toBe(false);
```
Comment (flipactual) [inline __tests__/Exciter.js:971]: Moved `null` to the other list here

**Review by chasingmaxwell — APPROVED** [review]
Looks good! Thanks for this. Want to add yourself to the contributors prop in package.json?

**Pull request merged**